builder/builder_test.go

```diff
@@ -333,12 +338,9 @@ func getAppHashFromEVM(ethState *state.StateDB, header *monomer.Header) (common.
 }
 
 func TestWithdrawal(t *testing.T) {
```

Comment:
nit: Could `TestWithdrawal` be renamed to `TestBuildRollupTxs` and moved below the `TestBuild` test for readability?